ecommerce-webhosting/terraform/gcs.tf

```
@@ -0,0 +1,53 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+resource "random_id" "bucket_prefix" {
+  byte_length = 8
+}
+
+resource "google_storage_bucket" "static_website" {
+  name          = "${random_id.bucket_prefix.hex}-static-website-bucket"
+  location      = "US"
+  storage_class = "COLDLINE"
```

```suggestion
  storage_class = "STANDARD"
```
Can you make it Standard to avoid 90-day retention period for Coldline?